### ISSUE TITLE
why always appear 'inf' and 'nan' when using 'polygraphy run'

### ISSUE THREAD

**Issue by AlexHex7** [NVIDIA/TensorRT#2724]
## Environment

**TensorRT Version**: 8.5.3.1
**NVIDIA GPU**:  GeForce RTX 3090
**NVIDIA Driver Version**: 
**CUDA Version**:  11.8
**CUDNN Version**: 8.6.0.163
**Operating System**:  Ubuntu 18.04
**Python Version (if applicable)**:  3.8
**Tensorflow Version (if applicable)**: x
**PyTorch Version (if applicable)**:  1.9.0
**Baremetal or Container (if so, version)**: 

## Code

~~~
# demo.py
import torch
from torch import nn


class Network(nn.Module):
    def __init__(self):
        super(Network, self).__init__()

        self.layers = nn.Sequential(
            nn.Conv3d(1, 16, 3, 1, 1),
            nn.Conv3d(16, 16, 3, 1, 1),
        )

    def forward(self, x):
        return self.layers(x)


if __name__ == '__main__':
    net = Network()
    net.cuda()

    torch.set_grad_enabled(False)
    net.eval()

    x = torch.randn(1, 1, 128, 224, 224).cuda()
    torch.onnx.export(
        model=net,
        opset_version=11,
        args=x,
        f='demo.onnx',
        
        input_names=['input1'],
        output_names=['output1'],
        dynamic_axes={'input1': {0: 'batch_size'}},
    )

~~~

## Command
~~~
python demo.py
polygraphy run demo.onnx --trt --fp16 --onnxrt --trt-outputs mark all --onnx-outputs mark all
~~~

## Results
~~~
[I] Creating ONNX-Runtime Inference Session with providers: ['CPUExecutionProvider']
[I] onnxrt-runner-N0-03/02/23-14:39:54 
    ---- Inference Input(s) ----
    {input1 [dtype=float32, shape=(1, 1, 128, 224, 224)]}
[I] onnxrt-runner-N0-03/02/23-14:39:54 
    ---- Inference Output(s) ----
    {5 [dtype=float32, shape=(1, 16, 128, 224, 224)],
     output1 [dtype=float32, shape=(1, 16, 128, 224, 224)]}
[I] onnxrt-runner-N0-03/02/23-14:39:54  | Completed 1 iteration(s) in 424.4 ms | Average inference time: 424.4 ms.
[I] Accuracy Comparison | trt-runner-N0-03/02/23-14:39:54 vs. onnxrt-runner-N0-03/02/23-14:39:54
[I]     Comparing Output: '5' (dtype=float32, shape=(1, 16, 128, 224, 224)) with '5' (dtype=float32, shape=(1, 16, 128, 224, 224))
[I]         Tolerance: [abs=1e-05, rel=1e-05] | Checking elemwise error
[I]         trt-runner-N0-03/02/23-14:39:54: 5 | Stats: mean=0.013723, std-dev=0.35261, var=0.12434, median=-0.056522, min=-1.1596 at (0, 5, 52, 84, 86), max=1.475 at (0, 3, 113, 25, 23), avg-magnitude=0.28514
[I]         onnxrt-runner-N0-03/02/23-14:39:54: 5 | Stats: mean=0.013723, std-dev=0.35261, var=0.12434, median=-0.056522, min=-1.1596 at (0, 5, 52, 84, 86), max=1.475 at (0, 3, 113, 25, 23), avg-magnitude=0.28514
[I]         Error Metrics: 5
[I]             Minimum Required Tolerance: elemwise error | [abs=2.3842e-07] OR [rel=nan] (requirements may be lower if both abs/rel tolerances are set)
[I]             Absolute Difference | Stats: mean=2.8055e-10, std-dev=3.0849e-09, var=9.5167e-18, median=0, min=0 at (0, 0, 0, 0, 0), max=2.3842e-07 at (0, 8, 97, 216, 127), avg-magnitude=2.8055e-10
[I]             Relative Difference | Stats: mean=nan, std-dev=nan, var=nan, median=nan, min=nan at (0, 0, 91, 187, 196), max=nan at (0, 0, 91, 187, 196), avg-magnitude=nan
[I]         PASSED | Output: '5' | Difference is within tolerance (rel=1e-05, abs=1e-05)
[I]     Comparing Output: 'output1' (dtype=float32, shape=(1, 16, 128, 224, 224)) with 'output1' (dtype=float32, shape=(1, 16, 128, 224, 224))
[I]         Tolerance: [abs=1e-05, rel=1e-05] | Checking elemwise error
[I]         trt-runner-N0-03/02/23-14:39:54: output1 | Stats: mean=-0.021679, std-dev=0.21794, var=0.047499, median=0.018356, min=-0.95752 at (0, 3, 79, 65, 212), max=0.70996 at (0, 10, 1, 185, 182), avg-magnitude=0.17289
[I]             ---- Histogram ----
                Bin Range        |  Num Elems | Visualization
                (-0.958, -0.791) |       1486 | 
                (-0.791, -0.624) |     220106 | 
                (-0.624, -0.457) |    3828645 | ####
                (-0.457, -0.29 ) |   11639145 | ##############
                (-0.29 , -0.124) |   12543780 | ###############
                (-0.124, 0.0432) |   28591403 | ##################################
                (0.0432, 0.21  ) |   32927248 | ########################################
                (0.21  , 0.377 ) |   12360578 | ###############
                (0.377 , 0.544 ) |     646294 | 
                (0.544 , 0.71  ) |       1763 | 
[I]         onnxrt-runner-N0-03/02/23-14:39:54: output1 | Stats: mean=-0.021684, std-dev=0.21794, var=0.0475, median=0.018352, min=-0.95771 at (0, 3, 79, 65, 212), max=0.71048 at (0, 10, 1, 185, 182), avg-magnitude=0.17289
[I]             ---- Histogram ----
                Bin Range        |  Num Elems | Visualization
                (-0.958, -0.791) |       1478 | 
                (-0.791, -0.624) |     221029 | 
                (-0.624, -0.457) |    3823314 | ####
                (-0.457, -0.29 ) |   11639961 | ##############
                (-0.29 , -0.124) |   12549403 | ###############
                (-0.124, 0.0432) |   28591530 | ##################################
                (0.0432, 0.21  ) |   32921818 | ########################################
                (0.21  , 0.377 ) |   12364123 | ###############
                (0.377 , 0.544 ) |     646024 | 
                (0.544 , 0.71  ) |       1768 | 
[I]         Error Metrics: output1
[I]             Minimum Required Tolerance: elemwise error | [abs=0.0034412] OR [rel=inf] (requirements may be lower if both abs/rel tolerances are set)
[I]             Absolute Difference | Stats: mean=0.00016008, std-dev=0.00015473, var=2.3942e-08, median=0.00011694, min=0 at (0, 0, 2, 73, 121), max=0.0034412 at (0, 3, 97, 201, 34), avg-magnitude=0.00016008
[I]                 ---- Histogram ----
                    Bin Range            |  Num Elems | Visualization
                    (0       , 0.000344) |   92078410 | ########################################
                    (0.000344, 0.000688) |    9344367 | ####
                    (0.000688, 0.00103 ) |    1138543 | 
                    (0.00103 , 0.00138 ) |     165326 | 
                    (0.00138 , 0.00172 ) |      27294 | 
                    (0.00172 , 0.00206 ) |       5245 | 
                    (0.00206 , 0.00241 ) |       1051 | 
                    (0.00241 , 0.00275 ) |        176 | 
                    (0.00275 , 0.0031  ) |         29 | 
                    (0.0031  , 0.00344 ) |          7 | 
[I]             Relative Difference | Stats: mean=inf, std-dev=nan, var=nan, median=0.00090353, min=0 at (0, 0, 2, 73, 121), max=inf at (0, 10, 9, 47, 58), avg-magnitude=inf
[I]                 
[E]         FAILED | Output: 'output1' | Difference exceeds tolerance (rel=1e-05, abs=1e-05)
[E]     FAILED | Mismatched outputs: ['output1']
[E] Accuracy Summary | trt-runner-N0-03/02/23-14:39:54 vs. onnxrt-runner-N0-03/02/23-14:39:54 | Passed: 0/1 iterations | Pass Rate: 0.0%
~~~
